src/Components/LoginPage/LoginPageComponents/FooterBlock.jsx

```diff
@@ -0,0 +1,28 @@
+import {Link} from "react-router-dom";
+import React from "react";
+import {SIGN_IN_PATH, SIGN_UP_PATH} from "../path";
+
+export const FooterBlock = (props) => {
+    let footerText = ''
+    let footerLink = ''
+    let footerLinkText = ''
+
+    switch (props.link) {
+        case SIGN_IN_PATH:
+            footerText = `Don't have an account?`
+            footerLink = SIGN_UP_PATH
+            footerLinkText = 'Sign Up'
+            break
+        case SIGN_UP_PATH:
+            footerText = `Already have an account?`
+            footerLink = SIGN_IN_PATH
+            footerLinkText = 'Sign In'
```

Comment:
Can be moved to constants in object with SIGN_IN_PATH or SIGN_UP_PATH key and extracting like:
`footerData[props.link]`